### DISCUSSION TITLE
I keep getting Invalid `prisma.product.findMany()` invocation: error

### DISCUSSION THREAD

**Discussion by colleyyyy** [prisma/prisma#24785] [Q&A]
### Question

I keep getting Invalid `prisma.product.findMany()` invocation: error.
sometime i have to restart supabase project to make this error.
`Invalid `prisma.product.findMany()` invocation:

```
Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(PostgresError { code: "26000", message: "prepared statement \"s0\" does not exist", severity: "ERROR", detail: None, column: None, hint: None }), transient: false })`
```
please help..

`
 "@prisma/client": "^5.16.2",
 "prisma": "^5.16.2",
`

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @colleyyyy 👋 

Thank you for raising this question.

Can you append `?pgbouncer=true` to the end of your Supabase pooled connection string to see if this error disappears? See this previous [discussion](https://github.com/prisma/prisma/discussions/22837) .

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
